stable/external-dns/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.16.3
+version: 2.16.4
```

Comment:
Please bump minor version instead

Reply:
I did this in my second commit.

Reply:
Never mind this comment, this will get fixed when I rebase.

Reply:
This has been fixed (it may create another merge conflict if this doesn't get merged quickly)

Reply:
This has been corrected.